src/EmailLexer.php

```diff
@@ -144,13 +144,6 @@ class EmailLexer extends AbstractLexer
      */
     public Token $current;
 
-    /**
-     * The next token in the input.
-     *
-     * @var Token<int, string>|null
-     */
-    public Token|null $lookahead;
```

Comment:
This property exists on the parent class and we don't need to override it. The problem is that in Lexer 2, there is not property type declared while on Lexer 3 there is.